### PR TITLE
Move contribution guides to the documentation

### PR DESCRIPTION
**Pull request by elegaanz**
To have everything in one place, and to take advantage of the in-context translation of the documentation.

Once this will be merged, I will make contribute.joinplu.me redirect here.

### CODE REVIEW

File: source/contribute/discussion.html.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+icon: users
+summary: 'Join our Matrix room, or our Loomio to take part to discussions about Plume, share your feedback, or ask for help.'
+time: 10 minutes
+---
+
+Discussion about Plume mostly take place on two websites: Loomio (long posts, more like a forum with decision tools) and Matrix
+(instant chat, more suited for support or feedback).
+
+Most of the discussions are in English, but if you are not fluent enough, you can speak in another language your are more comfortable
+with, maybe we speak it too, and even if we don't we will find ways to talk with you, don't worry!
```
Comment (igalic) [inline source/contribute/discussion.html.md:12]: 💜💜💜

**Review by igalic — COMMENTED** [review]
💜💜💜

**Comment by elegaanz** [comment]
Does anybody still want to (proof)read these guides, or do you think I can merge them as they are?

File: source/contribute/bug-report.html.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Then, all you have to do is to fill the title and the description of your issue, and to validate.
+
+If your issue isn't about Plume itself, find the appropriate repository [in the list](https://github.com/Plume-org), go to the "Issues"
+tab, click the green "New issue" button, and describe your problem.
+
+## If you don't have a GitHub account
+
+If you don't have a GitHub account, and don't want to create one, you can also report your issue on Matrix.
+Join the Plume room as explained in [this guide](/contribute/discussion), and send a message explaining what is wrong.
+We will take of creating an issue on GitHub for you.
```
Comment (elegaanz) [inline source/contribute/bug-report.html.md]: note to myself: take care

**Pull request merged**